### PR TITLE
Feature/tao 3398 stimulus key navigation

### PR DESCRIPTION
**Pull request by ssipasseuth**
* Renamed the responsesAccess plugin into the more generic keyNavigation plugin
* Added img stimulus navigation feature and highlight styles

### CODE REVIEW

File: views/js/runner/plugins/content/accessibility/keyNavigation.js [hunk 1]
```diff
@@ -137,40 +147,41 @@ define([
...
+                    count = focusables.length || 1;
                     cursor = null;
                     self.enable();
                 })
                 .on('unloaditem', function () {
                     self.disable();
                 })
-                .on('previous-response', function() {
+                .on('previous-focusable', function() {
+                    console.log('aaa');
```
Comment (jsconan) [inline views/js/runner/plugins/content/accessibility/keyNavigation.js]: Forgotten debug line...

File: views/js/runner/plugins/content/accessibility/keyNavigation.js [hunk 2]
```diff
@@ -137,40 +147,41 @@ define([
...
         },
 
         /**
          * Called during the runner's render phase
          */
-        render: function render() {
-            var $container = this.getAreaBroker().getToolboxArea();
-            $container.append(this.$button);
-        },
+        render: _.noop,
```
Comment (jsconan) [inline views/js/runner/plugins/content/accessibility/keyNavigation.js]: Not sure this is needed. You can just remove the declaration of this method. If it is missing this won't trigger any error as the call will simply be ignored (delegation).

File: views/js/runner/plugins/content/accessibility/keyNavigation.js (hunk 1 above)
Comment (ssipasseuth) [inline views/js/runner/plugins/content/accessibility/keyNavigation.js]: hooops

File: views/js/runner/plugins/content/accessibility/keyNavigation.js (hunk 2 above)
Comment (ssipasseuth) [inline views/js/runner/plugins/content/accessibility/keyNavigation.js]: ok, thanks

**Comment by ssipasseuth** [comment]
ok, I removed unwanted codes

**Pull request merged**